### ISSUE TITLE
Question about targets names and DUD-E dataset

### ISSUE THREAD

**Issue by vincrichard** [dockstring/dockstring#166]
Dear authors, thank you for your work.

I looked into the various targets you shared and compared them with the DUD-E dataset. There are a couple of targets that I can't seem to find in the DUD dataset. 
I found the DUD-E dataset here: https://dude.docking.org/targets

Dockstring targets that don't match names from DUD-E:
```
{'ache',
 'adam17',
 'adora2a',
 'ar',
 'ca2',
 'cyp2c9',
 'cyp3a4',
 'dhfr',
 'drd2',
 'f10',
 'f2',
 'gba',
 'hmgcr',
 'hsd11b1',
 'hsp90aa1',
 'kdr',
 'maob',
 'map2k1',
 'mapk1',
 'mapk14',
 'mapkapk2',
 'nr3c1',
 'pgr',
 'ptgs2',
 'ptk2',
 'ptpn1',
 'ren'}
```

Do you mind sharing more info about this? From the paper all the targets came from DUD-E so I am wondering if it's just another appellation for the same target?

**Comment by vincrichard** [comment]
I did a little bit more research especially by using the 3d position to see if some file in the dude target at the same position. 
So I get the following naming for dockstring and DUD-E.

Dockstring -> DUD-E
ache -> aces
adam17 -> ada17
adora2a -> aa2ar
ar ->  andr
ca2 -> cah2
cyp2c9 -> cp2c9
cyp3a4 -> cp3a4
dhfr -> dyr
drd2 (added in dockstring)
f10 -> fa10
f2 -> thrb
gba -> glcm
hmgcr -> hmdh
hsd11b1 -> dhi1
hsp90aa1 -> hs90a
kdr -> vgfr2
maob -> aofb
map2k1 -> mp2k1
mapk1 -> mk01
mapk14 -> mk4
mapkapk2 -> mapk2
nr3c1 -> gcr
pgr -> prgr
ptgs2 -> pgh2
ptk2 -> fak1
ptpn1 -> ptn1
ren -> reni


I am not an expert on protein, but is there any reason for having a different naming convention ?

**Comment by mgarort** [comment]
Hi @vincrichard . You are right: the naming convention in DOCKSTRING is different from that of DUD-E. The reason is that we follow the names in the [ExCAPE database](https://jcheminf.biomedcentral.com/articles/10.1186/s13321-017-0203-5), which in turn follows the names from [HUGO nomenclature](https://en.wikipedia.org/wiki/HUGO_Gene_Nomenclature_Committee).

We took the protein structures from DUD-E and the ligands from ExCAPE, and we made sure to include active ligands for each target (i.e. active ligands according to the experimental activities recorded in ExCAPE, not to the docking scores produced by Vina; the two should be correlated for a certain protein target if Vina is doing a good job for that target, but this may not be the case in every target). We needed to translate the names in DUD-E (which were not following a single convention as far as I know) to ExCAPE (which was following a well-recognized convention, the HUGO nomenclature) in order to make sense of the two datasets together.

Hope that makes it clear!

By the way I have checked the conversions you've done and I they are correct.
